### PR TITLE
Created About Page

### PR DESCRIPTION
**Pull request by vipulchodankar**
Need other members' images, content to replace placeholders and some feedback.

### CODE REVIEW

**Review by kevinam99 — COMMENTED** [review]
The design is good. However, the pictures don't seem to load at https://whiplash7.github.io/Project-Oslo/about/

Please check this.

**Comment by vipulchodankar** [comment]
That's an issue with the project url in my fork.
It'll work with the main repo, since it starts at the root.

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous" defer></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous" defer></script>
+        <script src="script.js" type="text/javascript" async></script>
+        <script src="scrollreveal/dist/scrollreveal.min.js" defer></script>
+    </head>
+
+    <body>
+
+        <!-- Back to top button-->
```
Comment (JadeMaveric) [inline about/index.html]: Unless you plan on animating the scroll, using an anchor would be a better approach.

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+                                <p class="card-text">PLACEHOLDER TEXT<br>Established on ... <br>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minus nam provident ab. Magnam, necessitatibus ratione?</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            <div class="col-md-9 office">
+                <div class="row justify-content-around my-2 mb-5">
+                    <div class="col-md-5 col-lg-3">
+                        <img src="/about/imgs/siyakamat.jpg" class="img-fluid" alt="placeholder 1">
```
Comment (JadeMaveric) [inline about/index.html:123]: This file is HUGE!!! Crop and compress it.

**Review by JadeMaveric — CHANGES_REQUESTED** [review]
1. about/imgs/siyakamat.jpg is surprisingly huge (973 KB), It should be cropped and shrunk
2. Unless you plan on animating it (which I think would be great) use vanilla HTML to scroll to top.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+        <script src="scrollreveal/dist/scrollreveal.min.js" defer></script>
+    </head>
+
+    <body>
+
+        <!-- Back to top button-->
+        <button onclick="topFunction()" id="top" title="Back to top">
+            <i style="size: 22px;" class="fa fa-angle-up"></i>
+        </button>
+        <!--Back to top ends-->
```
Comment (JadeMaveric) [inline about/index.html:33]: Unless you plan on animating the scroll, using an anchor would be a better approach.

File: about/index.html (hunk 2 above)
Comment (kevinam99) [reply about/index.html:123]: If cropping can't be possible, just compress it. https://tinyjpg.com/ does the work well @whiplash7 and @JadeMaveric .

File: about/index.html (hunk 3 above)
Comment (kevinam99) [reply about/index.html:33]: @JadeMaveric , please propose the relevant changes for animating the scroll.

**Comment by vipulchodankar** [comment]
I haven't worked with the scrolling. Have just used the template. Regarding filesize, need to crop all images in 1:1 ratio and optimise them. Waiting for all the images.

**Comment by JadeMaveric** [comment]
@whiplash7 which images do you need? Make an issue with a checklist and <content request>. I'll get to asking them after that.

File: about/index.html (hunk 3 above)
Comment (JadeMaveric) [reply about/index.html:33]: Done. 6404e2c8d33f3c2383f5196cd199170de44c585f

**Comment by vipulchodankar** [comment]
I'm gonna redo the whole about page.

**Pull request closed without merging**